relayer/workers/parachaincommitmentrelayer/ethereum-channel-writer.go

```diff
@@ -0,0 +1,137 @@
+package parachaincommitmentrelayer
+
+import (
+	"context"
+	"fmt"
+
+	"golang.org/x/sync/errgroup"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/sirupsen/logrus"
+
+	"github.com/snowfork/polkadot-ethereum/relayer/chain"
+	"github.com/snowfork/polkadot-ethereum/relayer/chain/ethereum"
+	"github.com/snowfork/polkadot-ethereum/relayer/contracts/inbound"
+	"github.com/snowfork/polkadot-ethereum/relayer/substrate"
+)
+
+type EthereumChannelWriter struct {
```

Comment:
Can we shorten the name of the file to `writer.go` and the name of this struct to `Writer`? Its already namespaced so there won't be any symbol clashes.